server/protocols/jwt/src/main/java/org/apache/james/jwt/JwtTokenVerifier.java

```diff
@@ -60,6 +62,19 @@ public boolean verify(String token) {
         }
     }
 
+    public Optional<String> verifyAndExtractLogin(String token) {
+        try {
```

Comment:
just curious but is [verify](https://github.com/apache/james-project/pull/474/commits/7fa7f7d10e70d70427e61c6eb2bdc0c6471dc3e0#diff-873db7c453d2864562c7bcb918d1ecea44a765268b48c079353e4144c18f1f6aR52) still useful ?

 it seems it could be implemented as `verifyAndExtractLogin().isPresent()`